libnullpay/src/utils/rand.rs

```diff
@@ -1,6 +1,16 @@
-use rand;
-use rand::Rng;
+use std::iter;
+use rand::prelude::*;
+use rand::distributions::Alphanumeric;
+use sha2::Digest;
 
 pub fn get_rand_string(len: usize) -> String {
-    rand::thread_rng().gen_ascii_chars().take(len).collect()
-}
\ No newline at end of file
+    let mut rng = thread_rng();
+    iter::repeat(()).map(|()| rng.sample(Alphanumeric)).take(len).collect()
+}
+
+pub fn gen_rand_signature(address: &str, message: &[u8]) -> Vec<u8> {
```

Comment:
Should it be called like "hash_as_signature" and be moved out from rand.rs?

Reply:
Sure that’s a simple change